### PR TITLE
Assignment4 @alowe4

### PR DESCRIPTION
**Pull request by jdriscoll1**
Continuing building off of our version of website:
Link to the PR: https://github.com/jdriscoll1/EWU-CSCD379-2023-Spring/pull/14 
Link to the main website: https://happy-plant-0138e301e.3.azurestaticapps.net/
Link to the API: wordlewebspring2023.azurewebsites.net

### CODE REVIEW

File: wordle-web/src/components/NavigationBar.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <v-spacer></v-spacer>
+
+        <v-btn>
+          <SettingsDialogVue />
+        </v-btn>
+
+        <template v-slot:append>
+          <v-menu>
+            <template v-slot:activator="{ props }">
+              <v-btn v-bind="props" icon="mdi-hamburger"> </v-btn>
```
Comment (j0shh3ss) [inline wordle-web/src/components/NavigationBar.vue:23]: Is not visible, Background takes over

**Review by j0shh3ss — COMMENTED** [review]
Make all pages responsive ❌ (Cannot see the title bar)
1080p desktop ✅ (Nice background)
iPad Air ✅ (Nice title page)
Samsung Galaxy S20 Ultra Super Mega Excellent ✅ (I assume so?)
IPhone SE ✅ (Not sure but I assume so)
Create a improved landing page and site theme (Background, make it look attractive to play) ✅
Theme should extend all pages ✅
Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ✅
Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ❌ (Does not show the word but shows everything else)
It should indicate if the user has played that day ❌
Clicking on the item should take the player to that game (Extra Credit)

Looks good! Nice work.

**Comment by jdriscoll1** [comment]
Hi @j0shh3ss , thanks for taking a look. To follow up, under what circumstances is are you unable to see the title bar? Also, please clarify the condition in which it did not show that the player played a word. Thank you

File: wordle-web/src/components/NavigationBar.vue (hunk 1 above)
Comment (jdriscoll1) [reply wordle-web/src/components/NavigationBar.vue:23]: Hi Josh, does the background take over when you scroll up?

**Pull request closed without merging**